index.html

```diff
@@ -1,36 +1,233 @@
+        <title>Rominas Microverse Student Project Portfolio</title>
+        <link rel="stylesheet" href="style.css" />
+    </head>
+
 <body>
-    <header class="header">
-        <div class="header-container">
-            <div class="logo">
-            <a href="#">My logo</a>
-            </div>
-        <img class="menu" src="images/Union.svg" alt="menu">
-        </div>
-    </header>
-<section class="my-name">
-    <div class="name"><h1>I'm Romina! <br>Glad to see<br> you</h1></div> 
-    <div class="greetings"><p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></div>
-    <div class="connect"> LET'S CONNECT</div>
-    <div class="social-media">
-        <ul>
-            <li><img class="social-media twitter" src="images/twitter.svg" alt="twitter" /></li>  
-            <li><img class="social-media linkedin" src="images/linkedin.svg" alt="linkedin"/></li>
-            <li><img class="social-media github" src="images/github.svg" alt="github" /></li>
-            <li> <img class="social-media medium" src="images/medium.svg" alt="medium" /></li>
-            <li><img class="social-media medium" src="images/vector.svg" alt="vector" /></li>
+<!--HEADER SECTION-->
+<div class="background"></div>
+<header id="header">
+  <div class="contenedor">
+    <!--logo section-->
+    <div class="logo">
+      <a href="#">Romina </a>
+    </div>
+    <!--end of logo section-->
+    <!--barra de navegacion -->
+    <nav id="nav">
+      <div class="nav-content">
+        <ul class="nav-items">
+          <li class="nav-item">Portafolio</li>
+          <li class="nav-item">About</li>
+          <li class="nav-item">Contact</li>
         </ul>
+      </div>
+      <img class="menu" src="/images/menu.svg" alt="menu"/>
+    </nav>
+    <!--fin de barra de navegacion -->
+  </div>
+</header>
+<!--END OF THE HEADER SECTION-->
+
+<!--ID SECTION-->
+<main id="main">
+  <div class="contenedor">
+    <article class="main-content">
+      <h1 class="title"> I'm Romina, glad to see you!</h1>
+        <p class="main-description"> 
+        I'm a software developer! I can help you build a product ,feature or
+        website. Look through some of my work and experience! if you like
+        what you see and have a project you need coded, don't hesitate to
+        contact me.
+        </p>
+        <p class="connect">LETS CONNECT</p>
+        <div class="social-network">
+          <img src="images/twitter-logo.svg" alt="twitter" />
+          <img src="images/Linkedin.svg" alt="linkedin"/>
+          <img src="images/github.svg" alt="github"/>
+          <img src="images/Vector2.svg" alt="#"/>
+          <img src="images/Vector.svg" alt="#"/>
+        </div>
+    </article>
+  </div>
+</main>
+<!--END OF ID SECTION-->
+
+<!--WORK SECTIONS-->
+<section id="work">
```

Comment:
- [ ] Great work you have done with the work section, kindly add a bottom margin to the cards, so it aligns with the Figma Design

### Your Design!
![m2](https://user-images.githubusercontent.com/48371156/170811603-f8f24eff-25b0-41f6-a2f1-8511aa6730e0.png)\

### Template
![f2](https://user-images.githubusercontent.com/48371156/170811604-46068af1-24b1-4a21-9c22-3e4ba4871561.png)